java/spacewalk-java.changes

```diff
@@ -66,8 +67,8 @@
 - UI render without error if salt-formulas system folders are unreachable (bsc#1142309)
 - Add susemanager as prerequired for spacewalk-java
 - Cloning Errata from a specific channel should not take packages
-- Hide channels managed by Content Lifecycle projects from available sources (bsc#1137965)
   from other channels (bsc#1142764)
+- Hide channels managed by Content Lifecycle projects from available sources (bsc#1137965)
```

Comment:
This one is funny: This line was inserted (for some reason) between two lines of another changelog.